robottelo/cli/base.py

```diff
@@ -73,25 +73,30 @@ class Base:
 
     @classmethod
     def _handle_response(cls, response, ignore_stderr=None):
-        """Verify ``return_code`` of the CLI command.
+        """Verify ``status`` of the CLI command.
 
         Check for a non-zero return code or any stderr contents.
 
-        :param response: a ``SSHCommandResult`` object, returned by
-            :mod:`robottelo.ssh.command`.
+        :param response: a result object, returned by :mod:`robottelo.ssh.command`.
         :param ignore_stderr: indicates whether to throw a warning in logs if
             ``stderr`` is not empty.
         :returns: contents of ``stdout``.
         :raises robottelo.cli.base.CLIReturnCodeError: If return code is
             different from zero.
         """
-        if response.return_code != 0:
+        if isinstance(response.stderr, tuple):
+            # the current contents of response.stderr is a tuple with the following properties
+            # (<len(message)>,<message>).
+            # This behavior could (and maybe should) change in the near future.
+            # In the meantime, we don't need it here so we just use the message itself
+            response.stderr = response.stderr[1]
+        if response.status != 0:
             full_msg = (
                 f'Command "{cls.command_base} {cls.command_sub}" '
-                f'finished with return_code {response.return_code}\n'
+                f'finished with status {response.status}\n'
                 f'stderr contains:\n{response.stderr}'
```

Comment:
```suggestion
                f'stderr contains:\n{response.stderr}\n'
                f'stdout contains:\n{response.stdout}\n'
```
Can we also add `response.stdout`? sometimes  `response.stderr` and  `response.status` are not enough to debug the failure.

Reply:
This could go either way, but I don't want this PR to bleed into small enhancements outside the scope of the ssh module conversion and helper method elimination.

Reply:
okay